test/parallel/test-repl-domain.js

```diff
@@ -2,7 +2,7 @@
 var assert = require('assert');
 var common = require('../common');
 
-var util   = require('util');
+var util = require('util');
```

Comment:
Stylistic change, arguably doesn't belong in a commit with the word 'remove' in it.